### DISCUSSION TITLE
Replacement for PTZ Cameras (afordable and & available)

### DISCUSSION THREAD

**Discussion by ABrauser** [roleoroleo/yi-hack-Allwinner-v2#902] [Q&A]
Hi everyone,

I'm a huge fan of this project, especially since I bought the Yi Outdoor 1080p PTZ back in April 2021. Special thanks to @roleoroleo for integrating the new camera into the project back in the days.

I'm planning a new project where I need two outdoor PTZ cameras, but unfortunately, the Yi Outdoor 1080p PTZ is almost impossible to find / buy. Does anyone know of an alternative PTZ camera that works with this custom firmware? Alternatively, would it be difficult to integrate a PTZ camera similar to the Yi Camera products into this custom firmware?

Any advice or recommendations would be greatly appreciated!

Thank you!

PS: (I also posted this in yi-hack-Allwinner but i just noticed that v2 has more camera models.)

**Comment by roleoroleo** [comment]
Yi dome U are supported by allwinner-v2 project.
Available on Aliexpress.
The features are the same.
My h52ga works fine,
